upb/util/def_to_proto.c

```diff
@@ -392,10 +392,14 @@ static google_protobuf_FileDescriptorProto *filedef_toproto(
   google_protobuf_FileDescriptorProto_set_name(
       proto, strviewdup(ctx, upb_filedef_name(f)));
 
-  size_t n = strlen(upb_filedef_package(f));
-  if (n) {
-    google_protobuf_FileDescriptorProto_set_package(
-        proto, strviewdup(ctx, upb_filedef_package(f)));
+  const char* package = upb_filedef_package(f);
+  size_t n;
```

Comment:
the value of n from this block is not used, so I think you should embed the local into the block to keep its scope tighter

Reply:
Done. `n` is also used later in the function, but I moved the declaration down to the next usage.